pkg/features/kube_features.go

```diff
@@ -369,6 +369,14 @@ const (
 	// fallback to using it's cgroupDriver option.
 	KubeletCgroupDriverFromCRI featuregate.Feature = "KubeletCgroupDriverFromCRI"
 
+	// owner: @vinayakankugoyal
+	// kep: http://kep.k8s.io/2862
+	// alpha: v1.32
```

Comment:
cleanup: remove this line like https://github.com/kubernetes/kubernetes/blob/master/pkg/features/kube_features.go 
- as already in the versioned_kube_features.go.

Reply:
Thanks, Done!